packages/admin-ui/src/pages/system/components/Notifications/EthicalMetrics.tsx

```diff
@@ -91,46 +143,153 @@ export default function EthicalMetrics() {
+                <div>
+                  <ol>
+                    <li>Create a private channel in your telegram.</li>
+                    <li>
+                      Add dappnode bot{" "}
+                      <span className={theme === "dark" ? "dark" : "light"}>
+                        @ethicalMetricsAlerts_bot
+                      </span>{" "}
+                      to the channel as an administrator.
+                    </li>
+                    <li>
+                      Open telegram in a web browser{" "}
+                      <a href="https://web.telegram.org/">(Telegram web)</a> and
+                      open the channel.
+                    </li>
+                    <li>
+                      Copy the channel id from the url. The channel Id is the
+                      number of 13 digits that comes just after the{" "}
+                      <span className={theme === "dark" ? "dark" : "light"}>
+                        -
+                      </span>{" "}
+                      in the url. It always starts with{" "}
+                      <span className={theme === "dark" ? "dark" : "light"}>
+                        -100
+                      </span>
+                      . While coping it, make sure to include the{" "}
+                      <span className={theme === "dark" ? "dark" : "light"}>
+                        -
+                      </span>{" "}
+                      just before the number!
+                    </li>
+                    <li>
+                      Paste it in the Telegram Channel Id field and toggle the
+                      switch{" "}
+                      <span className={theme === "dark" ? "dark" : "light"}>
+                        ON
+                      </span>{" "}
+                      to start receiving notifications.
+                    </li>
+                  </ol>
+                </div>
+              </Accordion.Collapse>
+            </div>
+          </Accordion>
+          <br />
+          <Form.Label>
+            Ethical metrics notifications by telegram channel
+          </Form.Label>
+          <Input
+            placeholder="Telegram Channel Id"
+            isInvalid={tgChannelIdError}
+            value={tgChannelId}
+            onValueChange={setTgChannelId}
+            append={
+              ethicalMetricsOn ? (
+                <Button
+                  type="submit"
+                  className="register-button"
+                  onClick={() =>
+                    enableEthicalMetricsSync({ tgChannelIdValue: tgChannelId })
+                  }
+                  variant="dappnode"
+                  disabled={
+                    tgChannelId === "" ||
+                    tgChannelId === ethicalMetricsConfig.data.tgChannelId ||
+                    tgChannelIdError
+                  }
+                >
+                  Update
+                </Button>
+              ) : (
+                <></>
```

Comment:
Dont add an empty element, instead use above the operator `ethicalMetricsOn && ( )`

Reply:
TypeScript can have issues inferring the correct type. Using **undefined** instead of an empty fragment is often clearer in terms of intent maybe?